src/servicebroker-npm/src/NodeStreamMessageReader.ts

```diff
@@ -0,0 +1,31 @@
+import { Disposable, MessageReaderOptions, RAL, ReadableStreamMessageReader } from 'vscode-jsonrpc'
+
+export class NodeStreamMessageReader extends ReadableStreamMessageReader {
```

Comment:
The readers and writers are copied from `vscode-jsonrpc`

- https://github.com/microsoft/vscode-languageserver-node/blob/main/jsonrpc/src/node/main.ts#L157-L167
- https://github.com/microsoft/vscode-languageserver-node/blob/main/jsonrpc/src/node/ril.ts#L154-L155

Reply:
Thanks for the source links. Code comments with these links would be preferable to PR comments, since code comments will be discoverable when we look at this code later.